### ISSUE TITLE
This page is too technical

### ISSUE THREAD

**Issue by NeilHoskins** [google/WebFundamentals#7282]
**Page Affected:** https://developers.google.com/web/updates/2017/09/autoplay-policy-changes

**What needs to be done?**

The decision to disable autoplay on videos has broken most e-learning that has ever been made. If you can't set embedded video to start as soon as the slide starts, you lose the synchronisation between the video and anything that might be happening on the slide.

This page of documentation appears to give different approaches to allowing autoplay to happen, but not in a way that most e-learning developers would understand. Most of us can copy & paste embed code and even have a rough understanding of what it does, but we're not web developers. If we were, we'd call ourselves "web developers" rather than "e-learning developers".

**Comment by LiveChief** [comment]
i am literally pissed off, gtfo 
this is a developer community no shit things are complicated... boo hoo go fucking learn.

**Comment by edwardchanjw** [comment]
@NeilHoskins doesn't teach you user/student to add them as PWA would worked, as mention? Probably Chrome love to have people who good at how to manipulate "Chrome standard" to worked around this issue. They like making a standard...They are making because they are mainstream and they really believe their philosophy.

Not a bad thing, but I would like:
1) Chrome extension / Apps would allow autoplay in permission settings
2) "Maybe just manual media-engagement, from user permission", The mute could just implemented on the Mute API over Chrome's Tab. But that get in the way of their philosophy of creating "Great Standard" and others browser vendor would follow, it is hard to follow, if they dont have Tab's mute like Chrome, different behavior across browser.
3) Keep following low disk space.....chrome://flags/#media-history and chrome://media-engagement/
